internal/server/errors.go

```diff
@@ -30,33 +31,30 @@ func (a *API) sendAPIError(c *gin.Context, err error) {
 	switch {
 	case errors.Is(err, internal.ErrUnauthorized):
 		resp.Code = http.StatusUnauthorized
-		resp.Message = "unauthorized"
+		// no messages for this type of error, it may contain sensitive information
```

Comment:
seems like a hard-coded message here was not harmful

Reply:
We should have some way of providing better error messages for these types. I think we should define an error type for forbidden and unauthorized. We would only create the type with a safe and sanitized message, which lets us include it here.  Any errors that are simply wrapped with a sentinel value will continue to have no response.

Reply:
Do we need any message at all for these? Usually we don't want to leak any info for authn/authz problems, and leave details in logs.

Reply:
I'm not sure. I think there are times when an authenticated request could benefit from more details about the authorization failure, without it being a security risk. I guess we don't have the problem right now, so we don't have to worry about it.